rtl/eclrtl/rtlread.cpp

```diff
@@ -171,7 +173,6 @@ void CThorStreamDeserializerSource::skipPackedInt()
 
 void CThorStreamDeserializerSource::skipUtf8(size32_t len)
 {
-    throwUnexpected();
```

Comment:
Based on the comments in the definition and the implementations of other skipXXX members of this class, the throwUnexpected() is correct (perhaps the dead code below it should be removed though)